src/Particle/Walker.h

```diff
@@ -161,10 +178,11 @@ class Walker
   }
 
   Walker(const Walker& a) : Properties(1, WP::NUMPROPERTIES, 1, WP::MAXPROPERTIES) { makeCopy(a); }
-  Walker(const Walker& a, long walker_id, long parent_id)
-      : walker_id_(walker_id), parent_id_(parent_id), Properties(1, WP::NUMPROPERTIES, 1, WP::MAXPROPERTIES)
+  Walker(const Walker& a, long walker_id, long parent_id) : Properties(1, WP::NUMPROPERTIES, 1, WP::MAXPROPERTIES)
   {
     makeCopy(a);
+    walker_id_ = walker_id;
```

Comment:
Here I make these assignments after makeCopy, putting them above means they get replace by `a`'s walker_id and parent_id which is not the behavior needed or expected.